config/initializers/filter_parameter_logging.rb

```diff
@@ -1,8 +1,8 @@
-# frozen_string_literal: true
-
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
+# Configure parameters to be filtered from the log file. Use this to limit dissemination of
+# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
+# notations and behaviors.
 Rails.application.config.filter_parameters += [
-  :password, :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
Why does the script remove `:password`, and do we want to re-add it?

Reply:
Yeah, that was a boo boo on my part. Fixed now. Seems like an odd choice. Happy to put it back in, even though we won't be passing passwords in the clear (certainly in this app)